src/modules/eventRegister/registration-status/useRegistrationStatus.ts

```diff
@@ -0,0 +1,14 @@
+import { computed, Ref } from "vue";
+export default function useRegistrationStatus(step: Ref<number>) {
+  const step1 = computed(() => {
+    return step.value === 1 || step.value === 2 || step.value === 3;
```

Comment:
question: is There a  case that step1 is false?
is step only range around 1-3?

Reply:
We always have step 1 and step only range from 1 to 3 krub